src/test/java/edu/unc/lib/boxc/migration/cdm/services/PermissionsServiceTest.java

```diff
@@ -193,20 +169,32 @@ public void generateDefaultPermissionsWithForceFlagTest() throws Exception {
         service.generatePermissions(options);
         assertTrue(Files.exists(permissionsMappingPath));
 
-        try (
-                Reader reader = Files.newBufferedReader(permissionsMappingPath);
-                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT
-                        .withFirstRecordAsHeader()
-                        .withHeader(PermissionsInfo.CSV_HEADERS)
-                        .withTrim());
-        ) {
-            List<CSVRecord> rows = csvParser.getRecords();
-            assertIterableEquals(Arrays.asList("default", "canViewMetadata", "canViewMetadata"), rows.get(0));
-        }
+        List<CSVRecord> rows = listCsvRecords(permissionsMappingPath);
+        assertIterableEquals(Arrays.asList("default", "canViewMetadata", "canViewMetadata"), rows.get(0));
+    }
+
+    @Test
+    public void generateWorkPermissionsWithDefaultTest() throws Exception {
+        testHelper.indexExportData("mini_gilmer");
+        Path permissionsMappingPath = project.getPermissionsPath();
+        var options = new PermissionMappingOptions();
+        options.setWithWorks(true);
+        options.setWithDefault(true);
+        options.setEveryone(UserRole.canViewMetadata);
+        options.setAuthenticated(UserRole.canViewMetadata);
+
+        service.generatePermissions(options);
+        assertTrue(Files.exists(permissionsMappingPath));
+
+        List<CSVRecord> rows = listCsvRecords(permissionsMappingPath);
+        assertIterableEquals(Arrays.asList("default", "canViewMetadata", "canViewMetadata"), rows.get(0));
+        assertIterableEquals(Arrays.asList("25", "canViewMetadata", "canViewMetadata"), rows.get(1));
+        assertIterableEquals(Arrays.asList("26", "canViewMetadata", "canViewMetadata"), rows.get(2));
+        assertIterableEquals(Arrays.asList("27", "canViewMetadata", "canViewMetadata"), rows.get(3));
     }
 
     @Test
-    public void generateWorkPermissionsTest() throws Exception {
+    public void generateWorkPermissionsGroupedWorksTest() throws Exception {
```

Comment:
Since grouping doesn't happen automatically you'll need to trigger it similar to how it's happening over in this other test:
https://github.com/UNC-Libraries/cdm-to-boxc-migration-util/blob/main/src/test/java/edu/unc/lib/boxc/migration/cdm/services/AggregateFileMappingServiceTest.java#L211